st2common/st2common/models/db/workflow.py

```diff
@@ -63,6 +63,7 @@ class TaskExecutionDB(stormbase.StormFoundationDB, stormbase.ChangeRevisionField
     task_name = me.StringField(required=True)
     task_id = me.StringField(required=True)
     task_spec = stormbase.EscapedDictField()
+    delay = me.IntField(min_value=0)
```

Comment:
Probably a good idea to also set the same ``min_value`` on ExecutionDB and LiveActionDB models.